testing/describes.js

```diff
@@ -518,6 +518,7 @@ class FakeWinFixture {
     if (this.spec.mockFetch !== false) {
       fetchMock./*OK*/restore();
     }
+    RequestBank.tearDown();
```

Comment:
we'd better wait for tearDown here to avoid race condition between server & client